test/languages/auto.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { newUnibeautify, Beautifier } from "unibeautify";
+import beautifier from "../../src";
+import options from "../../src/options";
+test(`should successfully beautify JavaScript through auto lang select`, () => {
+  const unibeautify = newUnibeautify();
+  unibeautify.loadBeautifier(beautifier);
+  beautifier.options.ActionScript = options.Script;
```

Comment:
I think this creates a side effect in the cached `beautifier`.

Maybe something like this instead:

```typescript
import beautifier from "../../src";
import options from "../../src/options";

const customBeautifier = {
  ...beautifier,
  options: {
    ActionScript: options.Script,
  }
};
unibeautify.loadBeautifier(customerBeautifier);
```

Reply:
You're testing this line of code, I suppose? https://github.com/Unibeautify/beautifier-prettydiff/blob/master/src/index.ts#L100

Reply:
Maybe choose a different language name instead of `ActionScript`, since it is a real language and I do not think this JavaScript code is it? Maybe `languageName="TestLanguage"` or something obvious?

Reply:
Would cloneDeep be better to create the beautifier to use?